gym_electric_motor/physical_systems/mechanical_loads/polynomial_static_load.py

```diff
@@ -47,18 +53,26 @@ def __init__(self, load_parameter=None, limits=None, load_initializer=None):
         self._c = self._load_parameter['c']
 
     def _static_load(self, omega, *_):
-        """
-        Calculation of the load torque for a given speed omega.
-        """
-        sign = 1 if omega > 0 else -1 if omega < 0 else 0
-        return sign * (self._c * omega**2 + self._b * abs(omega) + self._a)
+        """Calculation of the load torque for a given speed omega."""
+        sign = 1 if omega > 0 else -1 if omega < -0 else 0
```

Comment:
For ease of reading could we also use `sign = np.sign(omega)`?

Reply:
Yes it would be possible. However, it has shown that np.sign is quite slow on scalars compared to this if-else clause. And speed is important within the ODE methods.

![grafik](https://user-images.githubusercontent.com/50244395/134371033-1c1d68e5-e918-4db0-b40c-9ff6b9dfbb65.png)

Reply:
Good point, I didn't expect this to be the faster approach.